src/constants/tokens.ts

```diff
@@ -167,6 +159,23 @@ export const OHM_DAI_LP_TOKEN = new Token({
 OHM_DAI_LP_TOKEN.customPricingFunc = networkId =>
   calculateUniOrSushiLPValue({ networkId, lpToken: OHM_DAI_LP_TOKEN, poolTokens: [OHM_TOKEN, DAI_TOKEN] });
 
+export const OHM_DAI_ETH_LP_TOKEN = new Token({
+  decimals: 18,
+  name: "50OHM-25DAI-25WETH",
+  icons: ["OHM", "DAI", "ETH"],
+  factory: BalancerV2Pool__factory,
+  addresses: OHM_DAI_ETH_LP_ADDRESSES,
+  purchaseUrl: "https://app.balancer.fi/#/trade",
+});
+
+OHM_DAI_ETH_LP_TOKEN.customPricingFunc = networkId => {
+  return calculateBalancerLPValue({
+    networkId,
+    lpToken: OHM_DAI_ETH_LP_TOKEN,
+    poolTokens: [OHM_TOKEN, DAI_TOKEN, WETH_TOKEN],
+  });
+};
+
```

Comment:
This isn't really used right now, but I added the customPricingFunc here in the event we ever wanted to surface the LP token price in USD terms.